control-plane/pkg/reconciler/base/receiver_condition_set.go

```diff
@@ -183,6 +183,18 @@ func (manager *StatusConditionManager) TopicReady(topic string) {
 	)
 }
 
+func (manager *StatusConditionManager) FailedToGetBrokerAuthSecret(err error) reconciler.Event {
+
+	manager.Object.GetConditionSet().Manage(manager.Object.GetStatus()).MarkFalse(
+		apis.ConditionReady,
```

Comment:
We shouldn't use the ready condition directly but through other conditions, the ready condition will be marked true only when all other conditions are true, so we can reuse the `ConditionTopicReady` in this case.

Reply:
Ah ok. Didn't know that. I updated it...